Modules/_xxinterpchannelsmodule.c

```diff
@@ -1,13 +1,9 @@
 
 /* interpreters module */
 /* low-level access to interpreter primitives */
```

Comment:
Pretty sure this comment meant to apply to the definition of Py_BUILD_CORE_MODULE that was removed.

Reply:
The comment is describing the contents of the module rather than the ifdef.